### ISSUE TITLE
[Bug] Error Reported During Launch 

### ISSUE THREAD

**Issue by EldrinFal** [quiqueck/BetterNether#82]
### What happened?

[22:37:17] [Worker-Main-34/ERROR]: Couldn't load tag environmentz:warm_armor as it is missing following references: betternether:cincinnasite_helmet_chestplate (from Better Nether), betternether:cincinnasite_helmet_leggings (from Better Nether), betternether:cincinnasite_helmet_boots (from Better Nether)

Log
https://paste.ee/p/fUO7j

### BetterNether

7.1.1

### BCLib

2.1.2

### Fabric API

0.66.0

### Fabric Loader

0.14.10

### Minecraft

1.19.2

### Relevant log output

```shell
[22:37:17] [Worker-Main-34/ERROR]: Couldn't load tag environmentz:warm_armor as it is missing following references: betternether:cincinnasite_helmet_chestplate (from Better Nether), betternether:cincinnasite_helmet_leggings (from Better Nether), betternether:cincinnasite_helmet_boots (from Better Nether)
```

**Comment by quiqueck** [comment]
Hi, and thanks for reporting. I am not aware that we provide an item called `cincinnasite_helmet_chestplate`. That is why I think, that another mod or datapack tries to add those items to environmentz?

**Comment by quiqueck** [comment]
Ah, wait. This issue was fixed by this PR #72

**Comment by quiqueck** [comment]
Released with BetterNether 7.1.2. Please re-open this issue if you still have problems with this.
